Tasks/CopyFilesToMachines/CopyFilesToMachines.ps1

```diff
@@ -4,9 +4,7 @@ param (
     [string]$sourcePath,
```

Comment:
Should we start taking port as well in this task? We are hardcoding the port right now. Maybe open as a bug?